requirements.txt

```diff
@@ -4,8 +4,16 @@ pytest-cov
 pytest-runner
 build
 bumpversion
-freezegun
-pandas
-selenium
+freezegun~=1.2.2
+pandas~=2.0.1
+selenium~=4.9.1
 webdriver-manager
-sphinx
\ No newline at end of file
+sphinx
+requests~=2.28.1
+botocore~=1.27.96
+python-dateutil~=2.8.2
+boto3~=1.24.96
+xmltodict~=0.13.0
+python-dotenv~=1.0.0
+inflection~=0.5.1
+email-validator
```

Comment:
i actually think you could leave most of this out and instead update https://github.com/stac-labs/stac-utils-python/blob/main/setup.cfg - the python package installs requirements based on what's setup.cfg, not requirements.txt.

we do need to keep freezegun and sphinx because they are packages for testing and documentation, but the rest should be to set up the package